src/start/PasswordAuthScreen.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { TextInput } from 'react-native';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { fetchApiKey } from '../api/apiClient';
+import config from '../config';
+import { styles, Screen, ErrorMsg, Button } from '../common';
+import { getAuth } from '../accountlist/accountlistSelectors';
+import { loginSuccess } from '../account/accountActions';
+
+type Props = {};
+
+class PasswordAuthScreen extends React.Component {
+
+  props: Props;
+  state: {
+    progress: boolean,
+    email: string,
+    password: string,
+    error: string,
+  }
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      progress: false,
+      email: props.email || config.defaultLoginEmail,
+      password: props.password || config.defaultLoginPassword,
+    };
+  }
+
+  tryPasswordLogin = async () => {
+    const { auth } = this.props;
+    const { email, password } = this.state;
+
+    this.setState({ progress: true, error: '' });
```

Comment:
Can we make `error` null here? I think it's clearer if the no error case isn't an empty string